activemodel/lib/active_model/error.rb

```diff
@@ -130,15 +135,27 @@ def message
       end
     end
 
+    # Returns the error detail
+    #
+    #   error = ActiveModel::Error.new(person, :name, :too_short, count: 5)
+    #   error.detail
+    #   # => { error: :too_short, count: 5 }
```

Comment:
@vipulnsward  I think since I moved `messages` method here as `message` I did the same thing with `details`. But now I think about it it might not be grammatically correct. Would you think renaming it (or having an alias) is a good idea?